src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1259,6 +1260,18 @@ public static boolean loadAdvancements(Map<ResourceLocation, Advancement.Builder
         return errored;
     }
 
+    @Nullable
+    public static CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical, float damageModifier)
+    {
+        CriticalHitEvent hitResult = new CriticalHitEvent(player, target, damageModifier);
```

Comment:
I think he is saying `CriticalHitEvent` should take in `vanillaCritical` as well.